src/animator.js

```diff
@@ -351,6 +377,8 @@ export class VelocityAnimator {
       el.animations = {};
       el.animations.enter = this._parseAttributeValue(el.getAttribute('anim-enter')) || this.enterAnimation;
       el.animations.leave = this._parseAttributeValue(el.getAttribute('anim-leave')) || this.leaveAnimation;
+      el.animations.show = this._parseAttributeValue(el.getAttribute('anim-show')) || undefined;
```

Comment:
if the animation for leave/show is not defined shouldn't we use defaults like for enter/leave? See line 22/23